### ISSUE TITLE
Horizontal swiping on the html

### ISSUE THREAD

**Issue by VMBindraban** [ampproject/amphtml#25464]
**Please only file reports about bugs in AMP here.**

- If you have questions about how to use AMP or other general questions about AMP please ask them on Stack Overflow under the AMP HTML tag instead of filing an issue here: http://stackoverflow.com/questions/tagged/amp-html
- If you have questions/issues related to Google Search please ask them in Google's AMP forum instead of filing an issue here: https://goo.gl/utQ1KZ

If you have a bug for AMP please fill in the following template. Delete everything except the headers (including this text).

## What's the issue?
I don't know if this is intentional behaviour or not.

![cfdad33a-c091-41e1-9a36-ab40ef109fa0](https://user-images.githubusercontent.com/2269400/68298452-abc48200-0099-11ea-901d-b54b453996b0.png)

It's possible to swipe horizontal on every AMP site which makes it look that there is room on both sides of the page.

This is due the overflow visible on the HTML tag. (which can't be overridden)

## How do we reproduce the issue?

1. Go to https://amp.dev on mobile
2. Swipe horizontal

## What browsers are affected?

All mobile devices and maybe all touch devices

## Which AMP version is affected?
1911052130250

**Comment by kristoferbaxter** [comment]
Yes, this is intentional. When AMP documents are included in a Viewer, they may be swiped between document level units.

Horizontal swiping is only enabled for components that send a signal for canceling propagation to the Viewer.
